roles/chrony/tasks/install.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Install Chorny - Debian based distributions
+  tags: [chrony, chrony-install]
+  apt:
+    name: chrony
+    state: present
+    update_cache: yes
+    cache_valid_time: 3600
```

Comment:
You're updating apt cache if it's older than 1h, and you never update yum cache (default behaviour). These tasks should be as similar as possible.
Since yum module doesn't have `cache_valid_time`, I'd recommend to always update yum cache.